### PR TITLE
RMTEAM-311: Add integration tests

### PR DESCRIPTION
**Pull request by derlistigelurch**

### CODE REVIEW

File: TestProject/BadlyFormattedTests.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          {
+            MockRepository mockRepository = new MockRepository();
+            Page namingContainer = new Page();
+            IControl parentControlMock = mockRepository.PartialMock<IControl>();
+            IControl childControlMock = mockRepository.PartialMock<IControl>();
+
+            using (mockRepository.Ordered())
+            {
+              childControlMock.Expect (mock => PrivateInvoke.InvokeNonPublicMethod (mock, "OnInit", EventArgs.Empty));
+              parentControlMock.Expect (mock => PrivateInvoke.InvokeNonPublicMethod (mock, "OnInit", EventArgs.Empty));
```
Comment (iam-mholle) [inline TestProject/BadlyFormattedTests.cs]: I think this section should also be badly formatted to prove that the formatter works as intended and only fixes the formatting of rewritten nodes.

File: Core/RewriterOrchestrator.cs [hunk 2]
```diff
@@ -35,13 +35,15 @@ public static class RewriterOrchestrator
             new SetupResultForRewriter(),
             new ExpectCallRewriter(),
             new ConstraintsRewriter(),
+
```
Comment (iam-mholle) [inline Core/RewriterOrchestrator.cs]: Blank lines?

**Review by iam-mholle — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -16,4 +16,4 @@ jobs:
       run: dotnet build --configuration Release --no-restore
       
     - name: test
-      run: dotnet test --configuration Release --no-build --verbosity minimal
+      run: dotnet test Tests/Tests.csproj --configuration Release --no-build --verbosity minimal
```
Comment (iam-mholle) [inline .github/workflows/main.yml:19]: Why is this needed?

**Review by iam-mholle — APPROVED** [review]

File: .github/workflows/main.yml (hunk 3 above)
Comment (derlistigelurch) [reply .github/workflows/main.yml:19]: if i would run all tests, the integration test project would also run and the tests would fail, so i only run the tests in Tests.csproj
